modular_skyrat/modules/blueshield/code/blueshield.dm

```diff
@@ -71,12 +73,20 @@
 	uniform = /obj/item/clothing/under/plasmaman/blueshield
 
 /obj/item/storage/box/gunset/blueshield
-	name = "Allstar SC-3 PDW 'Hellfire' Gunset"
+	name = "CMG-1 Tactical Gunset"
```

Comment:
```suggestion
	name = "CMG-1 Gunset"
```
I don't think putting a single suppressor in a box automatically turns it tactical, it's the standard gun